tests/Directory.Build.props

```diff
@@ -5,6 +5,7 @@
   <PropertyGroup>
     <!-- Disable packaging and validation for test projects to fix benchmarks (that auto-generates boilerplate code) -->
     <IsPackable>false</IsPackable>
+    <BaseEnablePackageValidation>$(EnablePackageValidation)</BaseEnablePackageValidation>
```

Comment:
Package validation is re-enabled for the `Umbraco.Tests.Common` and `Umbraco.Tests.Integration` projects, so this stores the setting from the root `Directory.Build.props` in a new `BaseEnablePackageValidation` property, so they now follow the 'base' value...